app/assets/javascripts/govuk/selection-buttons.js

```diff
@@ -6,8 +6,6 @@
   if (typeof GOVUK === 'undefined') { root.GOVUK = {}; }
 
   var SelectionButtons = function (elmsOrSelector, opts) {
-    var $elms;
```

Comment:
@joelanman: I've just tested and it seems like this behaviour is already implemented, exactly as you describe it. So we can just remove the `app/assets/govuk/javascripts/selection-buttons.js` dupe from our codebase.